sql/core/src/test/scala/org/apache/spark/sql/MiscFunctionsSuite.scala

```diff
@@ -48,15 +48,16 @@ class MiscFunctionsSuite extends QueryTest with SharedSparkSession {
     checkAnswer(df.selectExpr("version()"), df.select(version()))
   }
 
-  test("SPARK-21957: get current_user in normal spark apps") {
+  test("SPARK-21957, SPARK-44860: get current_user, session_user in normal spark apps") {
```

Comment:
Can we also update `SPARK-21957: get current_user through thrift server`?

Reply:
Done